src/exometer_report.erl

```diff
@@ -622,6 +622,26 @@ new_entry(Entry) ->
 %%--------------------------------------------------------------------
 init([]) ->
     process_flag(trap_exit, true),
+    D = ets:foldl(
+        fun (#reporter{name = Name, module = Module, restart = Restart} = R, Acc) ->
+                terminate_reporter(R),
+                case add_restart(Restart) of
+                    {remove, How} ->
+                        case How of
+                            {M, F} when is_atom(M), is_atom(F) ->
+                                try M:F(Module, ?MODULE) catch _:_ -> ok end;
```

Comment:
Perhaps a more instructive reason? `?MODULE` is less descriptive than e.g. `parent_restart` would be